app/assets/javascripts/modules/repo-description.js

```diff
@@ -18,32 +20,58 @@ moj.Modules.repoDescription = {
   },
 
   bindEvents() {
-    this.$repoSlugSelect.on('change', () => {
-      if (this.$repoSlugSelect.val()) {
-        this.getRepoDescription();
-      } else {
-        this.updateDescription('');
-        $('#repo-results').addClass('js-hidden');
-      }
-    });
     this.$orgSelect.on('change', () => {
       this.showOrgRepos();
     });
   },
 
   showOrgRepos() {
     const currentOrg = this.$orgSelect.val();
+    const repoOptions = Array.from($(`optgroup[label="${currentOrg}"] option`));
+    const repos = repoOptions.map(opt => opt.text);
+    const descriptions = repoOptions.map(opt => opt.dataset.description);
+
+    // reset potentially pre-existing typeahead
```

Comment:
You may potentially move this in a function, something like `resetTypeahead()` which resets the typeahead component to its original state.

Reply:
I agree - wherever you have a comment, you can probably refactor into a self-documenting function

Reply:
👍